examples/nextjs-blog/components/header.js

```diff
@@ -45,17 +45,14 @@ const Header = () => {
         <nav>
           <ul>
             {mainNav.items.map(item => (
-              <li key={item.uuid}>
+              <li key={item.uuid} className={item.cssClass}>
```

Comment:
You'll have to move this to the separate Nav component now.

Reply:
I did

Reply:
👯

Reply:
Still seeing a conflict on this file though

Reply:
oh I see the other branch is merged in now